CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ x.y.z Release notes (yyyy-MM-dd)
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-swift/issues/????), since v?.?.?)
 * `@AutoOpen` and `@AsyncOpen` failed to use the `initialSubscriptions` set in
   the configuration passed to them ([PR #8572](https://github.com/realm/realm-swift/pull/8572), since v10.50.0).
+* Accessing `App.currentUser` from within a notification produced by `App.switchToUser()` 
+  (which includes notifications for a newly logged in user) would deadlock. 
+  ([Core #7670](https://github.com/realm/realm-core/issues/7670), since v14.6.0).
+* Inserting the same link to the same key in a dictionary more than once would incorrectly
+  create multiple backlinks to the object. This did not appear to cause any crashes later, but would 
+  have affecting explicit backlink count queries (eg: `$0.links.count`) and possibly notifications. 
+  ([Core #7676](https://github.com/realm/realm-core/issues/7676) since v14.5.2).
+* Automatic client reset recovery would crash when recovering on an `AnyRealmValue` property if its type was 
```

Comment:
This is omitting some nuance, which actually makes this point irrelevant for the Swift SDK - the crash was happening when `AddInteger` is called on a Mixed/AnyRealmValue property, but Swift doesn't support counters, so users shouldn't have been affected by it.